### ISSUE TITLE
Experience running on fly.io with 256MB

### ISSUE THREAD

**Issue by twnaing** [fspoettel/linkding-on-fly#7]
With 256MB RAM, there are frequent Out Of Memory (OOM) crashes. The idle RAM is about 220 MB. 

Without further configuration, import via html file is okay as long as 300 bookmarks per file. More than that and it crashed with OOM.

For the mass import, I run the linkding in local docker and do the import. Then backup sqlite file ([according to the documentation](https://github.com/sissbruecker/linkding/blob/master/docs/backup.md)) and replace the sqlite file on fly.io. I have to restart the fly.io. The existing user and user from local import both works.

I followed the fly.io documentation to [enable swap](https://fly.io/docs/rails/cookbooks/deploy/#enabling-swap) and it reduced idle RAM to 180MB.

**Comment by fspoettel** [comment]
hey @twnaing, thank you for the report. Do you think it would be reasonable to recommend 512MB as a base line or do you think the swap solution is enough?

**Comment by twnaing** [comment]
Even with swap solution, I still have OOM crashes. Except OOM mail from fly.io there are no loss of functions. 

I have 2 users but only 1 is active. The active user has hundreds of tags and almost 4000 bookarmks. I am using linkding and linkding injector extension on desktop firefox, and sharing shortcuts from android and iOS. When using via browser, quick successive navigation resulted in white blank page. However it worked when try again later.

fly.io mail mentions that if I do not want to see the OOM I can unsubscribe. 

In summary, 256 MB is workable with minor inconvenience.

**Comment by fspoettel** [comment]
one thing that significantly elevates memory usage is turning on the _Pre-load page information while browsing_ setting in the extension. turning that off resolved oom crashes for me in the past.

**Comment by fspoettel** [comment]
Bumped to 512MB required
